frontend/src/views/web/hosted-challenges.html

```diff
@@ -13,6 +13,8 @@
                         <span class=" ev-card-title fs-14"><span><img ng-src="{{challenge.image}}" ></span>{{challenge.title}}</span>
                     </div>
                     <div class="card-content">
+                        <li ng-repeat="tags in challenge.list_tags" class="list-inline-item chip orange white-text tag-size">{{tags}}</li>
```

Comment:
At some places it is `challenge.page.list_tags`, others it is `challenge.list_tags`. Just wanted some clarification on this.

Reply:
when we go inside the challenge that particular challenge data is stored inside the vm.page so those that are inside that page use that info